libs/errorHandler.js

```diff
@@ -3,18 +3,21 @@
 /*  Load gulp + plugins
 /* ------------------------------------------------------------------------- */
 
-var notify  = require("gulp-notify");
+var gutil = require('gulp-util');
```

Comment:
Nevermind. I originally thought it is possible to have various output formats of error object, and this file would handle all of them. As in the printing format. Maybe it covers all that already. Agree with having a nicer formatted output. Merging the changes.